.github/scripts/automergeCrowdinPr.js

```diff
@@ -12,8 +12,13 @@
 const CROWDIN_BRANCH = 'l10n/main'
 const CROWDIN_PR_USER = 'valora-bot-crowdin'
 
-const ALLOWED_UPDATED_FILE_MATCHER = new RegExp(`locales\/.*\/translation\.json`)
-const DISALLOWED_UPDATED_FILE = 'locales/base/translation.json'
+const ALLOWED_UPDATED_FILE_MATCHER = new RegExp(
+  `locales\/.*\/translation\.json|ios/celo\/.*\/InfoPlist.strings`
+)
+const DISALLOWED_UPDATED_FILES = [
+  'locales/base/translation.json',
+  'ios/celo/Base.lproj/InfoPlist.strings',
```

Comment:
@jeanregisser i also realised that these files were probably causing the last PR review to fail